boards/v2/qemu/qemu_arc/Kconfig.qemu_arc

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020 Synopsys, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_QEMU_ARC
+	bool
+
+config BOARD_QEMU_ARC_QEMU_EM4
+	bool
+	select BOARD_QEMU_ARC
+
+config BOARD_QEMU_ARC_QEMU_HS3X
+	bool
+	select BOARD_QEMU_ARC
+
+config BOARD_QEMU_ARC_QEMU_HS3X_XIP
+	bool
+	select BOARD_QEMU_ARC
+
+config BOARD_QEMU_ARC_QEMU_HS5X
+	bool
+	select BOARD_QEMU_ARC
+
+config BOARD_QEMU_ARC_QEMU_HS6X
+	bool
+	select BOARD_QEMU_ARC
```

Comment:
If moving the `select` to board level, then all those lines can be removed.
```suggestion
```

Reply:
Fixed